src/Arcus.Template.WebApi/Startup.cs

```diff
@@ -61,6 +78,15 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             #warning Please configure application with authentication mechanism: https://webapi.arcus-azure.net/features/security/auth/shared-access-key
 #endif
             app.UseMvc();
+
+//#if DEBUG
+            app.UseSwagger();
+            app.UseSwaggerUI(swaggerUiOptions =>
+            {
+                swaggerUiOptions.SwaggerEndpoint("v1/swagger.json", "Arcus.Template.WebApi");
+                swaggerUiOptions.DocumentTitle = "Arcus.Template.WebApi";
```

Comment:
Can we plug in the name of the project that was created based on this template?

Reply:
Yep, all occurences of the project name (`Arcus.Template.WebApi`) will be replaced by the name you give the created project.

Reply:
Awesome!